build-npm-module/index320.js

```diff
@@ -0,0 +1,9 @@
+const caseConverter = require('./index');
+
+const str = 'hello free Code Camp!';
+console.log(caseConverter.getUpperCase(str));
+console.log(caseConverter.getLowerCase(str));
+
+// Let's try your getProperCase function. Conver `str` to proper case and print it.
```

Comment:
Now let's try your `getProperCase()` function. Convert `str` to proper case and print it to the console.